warehouse/templates/manage/publishing.html

```diff
@@ -0,0 +1,129 @@
+{#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-#}
+
+{% extends "manage_project_base.html" %}
+
+{% set active_tab = 'publishing' %}
+
+{% block title %}
+  {% trans %}OIDC provider management{% endtrans %}
```

Comment:
We probably need to think up a more user-friendly way to refer to this feature, most users aren't going to know what OIDC is.

Reply:
Yeah, agreed.  We use the language of "publishing"/"publisher" elsewhere, WDYT about making that consistent? In that context, an OIDC provider becomes something like a "trusted publisher".

Reply:
I think "OpenID Connect Publisher" would probably be OK.